### DISCUSSION TITLE
本科生论文中英文封面日期问题

### DISCUSSION THREAD

**Discussion by FatCatCHC** [sjtug/SJTUThesis#964] [提问] [answered]
本科生毕设论文中英文封面中，默认的日期格式都为年月日，现在收到的格式审查意见要求我将中文封面上的格式改为年月，我在setup.tex内手动写了仅年月的日期，但是会导致英文封面也仅使用年月格式。请问有没有什么办法保持英文封面中的日期格式为原本的年月日而中文封面变为仅年月呢？

**Comment by LogCreative** [comment, marked as answer]
启用该配置。

https://github.com/sjtug/SJTUThesis/blob/8e403d73e1f2402e07c21db784d264c5b92b8753/setup.tex#L74

**Comment by FatCatCHC** [comment]
已解决，感谢
